src/style/style.css

```diff
@@ -0,0 +1,194 @@
+body {
```

Comment:
Need cleaning and formatting !
Remove everything that's not related to "global" styling ; the component specific styling should go to its specific stylesheet.